package.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "dmp-frontend",
-  "version": "0.0.1",
-  "description": "Digital Marketplace Frontend - Containing GOV.UK Frontned (Sass,JavaScrip, nunjucks templates) and Digital Marketplace custom components",
-  "main": "index.js",
+  "private": true,
+  "name": "digitalmarketplace-govuk-frontend-repository",
```

Comment:
Why `-repository`?

Reply:
Because this package.json is for the main "build tool" part of the digitalmarketplace-govuk-frontend. There will be a separate package.json in `/package` which will be added & used when we are ready to publish the code to NPM. We don't want to package up all the tooling code and dependencies in the packaged npm module

Reply:
Makes sense, thanks. This is all going to go in the README, right? 😸

Reply:
I'm working on the README ;)